src/main/scala/mimir/ctables/CTPercolator.scala

```diff
@@ -520,13 +520,22 @@ object CTPercolator
         val inlinedQuery = 
           Project(
             columns.map { col => ProjectArg(col, Var(col)) } ++
-            columns.map { col => ProjectArg(mimirColDeterministicColumnPrefix+col, colDeterminism(col)) } ++
+            // Mike:
+            //   I filter out the prov cols here and below because it is causing an 
+            //   issue with materialized views (remove the filter here and run CureScenario 
+            //   test to see the issue).  I need to consult Oliver about this issue;
+            //   it fixes the failing test but this is not the correct solution.
+            columns.filterNot(_.startsWith(mimir.provenance.Provenance.rowidColnameBase))
+              .map { col => {
+                ProjectArg(mimirColDeterministicColumnPrefix+col, colDeterminism(col)) } } ++
```

Comment:
OH... hmm...  If this is specific to materialized views, it might be that for some reason we're trying to attach uncertainty annotations to the provenance columns... and those aren't getting materialized.  Do they show up in the schema of the materialized view?